ktor-swagger/ktor-swagger.gradle.kts

```diff
@@ -1,4 +1,5 @@
 
 dependencies {
     implementation(group = "org.webjars", name = "swagger-ui", version = "3.1.5")
+    implementation(group = "org.apache.commons", name = "commons-lang3", version = "3.7")
```

Comment:
Not a fan of having to add this dependency but `TypeUtils.parameterize` simplified this code significantly.